dashboard/test/jobs/aichat_request_chat_completion_job_test.rb

```diff
@@ -0,0 +1,74 @@
+require "test_helper"
+
+class AichatRequestChatCompletionJobTest < ActiveJob::TestCase
+  setup do
+    @student = create :student
```

Comment:
Do you need these in `setup` since they're set in `factories`?

Reply:
Oh good catch! Yep can probably remove these - I can do so in a future PR.